Add possibility to specify loadBalancerIP address in load balancer type services (#1891)
strimzi/strimzi-kafka-operator: documentation/book/con-kafka-broker-external-listeners-loadbalancers.adoc

          external-dns.alpha.kubernetes.io/ttl: "60"
# ...
----

= Customizing the load balancer IP addresses

> On `loadbalancer` listeners, you can use the `loadBalancerIP` property to request a specific IP address for the load balancer which will be created.

Reviewer: ```suggestion
On `loadbalancer` listeners, you can use the `loadBalancerIP` property to request a specific IP address when creating a loadbalancer.
```

> This could be useful for example when you need to use a load balancer with a specific IP address.

Reviewer: ```suggestion
Use this property when you need to use a loadbalancer with a specific IP address.
```

> The `loadBalancerIP` field will be ignored if the cloud-provider does not support the feature.

Reviewer: ```suggestion
The `loadBalancerIP` field is ignored if the cloud provider does not support the feature.
```

> .Example of an external listener of type `loadbalancer` with specific load balancer IP address requests

Reviewer: ```suggestion
.Example of an external listener of type `loadbalancer` with specific loadbalancer IP address requests
```
